Refit.Tests/Refit.Tests.csproj

```diff
@@ -45,9 +44,15 @@
     <Message Text="Executing: $(RefitExecCmd)" Importance="High" />
     <Exec Command="$(RefitExecCmd)" />
     <Message Text="Processed Refit Stubs" />
+    <ItemGroup>
+      <Compile Condition="'$(TargetFramework)' == 'net461' " Include="RefitStubs.Net46.cs"
+               Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);$(BaseIntermediateOutputPath)**;$(BaseOutputPath)**;@(Compile)" />
```

Comment:
Are the Exclude's required as it's not a wildcard include?

Reply:
The MS documentation about MSBuild is quite poor. From what I guttered the Include in a target is incremental. I do not have the Full .Net on my PC and .Net Core have some different compartment on my setup and the Azure. So this staff I have to test on Azure only.
The RefitStubs.Net46.cs and RefitStubs.NetCore2.cs code at current are equal. So probably is good idea to clean the Stub generation a bit.